x-pack/plugins/endpoint/public/lib/saga/index.ts

```diff
@@ -4,15 +4,43 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import { AnyAction, Dispatch, Middleware, MiddlewareAPI } from 'redux';
+
+interface StoreAction extends AnyAction {
+  payload: unknown[];
+  type: string;
+}
+
+interface QueuedAction {
+  action: StoreAction;
+  state: unknown; // TODO: set to GlobalState once Pedro's commit is merged
```

Comment:
I think @peluja1012 is merged https://github.com/elastic/kibana/pull/51320

Reply:
done :)